### DISCUSSION TITLE
suggest compare with laravel page speed

### DISCUSSION THREAD

**Discussion by wafik** [abordage/html-min-benchmark#7] [Ideas]
hi i'am currently use this package **html min** with my laravel (8) projek, 
iam suggest for compare with laravel page speed thank's a lots

**Comment by abordage** [comment]
Hi, @wafik 
Are you talking about a package https://github.com/renatomarinho/laravel-page-speed?

I note that he has unclosed issues:
https://github.com/renatomarinho/laravel-page-speed/issues/165
https://github.com/renatomarinho/laravel-page-speed/issues/168

https://github.com/abordage/laravel-html-min works correctly with livewire and debug page
